src/Tests/EmbedButtonCrudTest.php

```diff
@@ -126,4 +127,47 @@ protected function deleteTests() {
     $this->assertTrue(empty($config), 'There are no embed buttons in config storage.');
   }
 
+  /**
+   * Tests the embed_button and file usage integration.
+   */
+  public function testEmbedButtonIcon() {
+    $this->enableModules(['system', 'user', 'file']);
+
+    $this->installSchema('file', ['file_usage']);
+    $this->installConfig(['system']);
+    $this->installEntitySchema('user');
+    $this->installEntitySchema('file');
+
+    $file1 = file_save_data(file_get_contents('core/misc/druplicon.png'));
+    $file1->setTemporary();
+    $file1->save();
+
+    $file2 = file_save_data(file_get_contents('core/misc/druplicon.png'));
+    $file2->setTemporary();
+    $file2->save();
+
+    $button = array(
+      'id' => 'test_button',
+      'label' => 'Testing embed button instance',
+      'button_label' => 'Test',
+      'entity_type' => 'node',
+      'entity_type_bundles' => array('article'),
+      'button_icon_uuid' => $file1->uuid(),
+      'display_plugins' => array('default'),
+    );
+
+    $entity = entity_create('embed_button', $button);
```

Comment:
The entity type has changed to 'entity_embed_button' now, and will need to be updated to pass tests.